guby_backend/views.py

```diff
@@ -39,7 +39,19 @@ def get_rolename(request, id):
 
 @login_required()
 def meetup_index(request):
-    meetups = Meetup.objects.filter(creator=request.user)
+
+    # get group names where the user is owner
+    group_names = request.user.groups.all()
+
+    # extract meetup id from group name
+    meetup_ids = []
```

Comment:
We can substitute this with list comprehensions instead? Something like 

```python
meetup_ids = [ group.name.split('-')[2] for group in group_names if g.name.startswith('guby-meetup') ]
```

Reply:
on second thought, can even refactor the split operation into a function for clearer reading?

```python
meetup_ids = [ get_meetup_id_from_group(group.name) for group in group_names if g.name.starswith('guby-meetup') ]
```

Reply:
@shrayasr moved to utils